osu.Game/Overlays/Profile/ProfileHeader.cs

```diff
@@ -539,6 +540,11 @@ private void load(OsuColour colours)
                     hoverColour = colours.Yellow;
                 }
             }
+
+            public class BrowserLinkText : LinkText, IHasTooltip

```

Comment:
call this `ProfileLink` and have it accept a user in the ctor. Then move all user-related sets to inside here.